tests/cbtest/gtest/test_export.cc

```diff
@@ -0,0 +1,267 @@
+#include <iostream>
+#include <libuzfs.h>
+#include <libzfs.h>
+#include <uzfs_mgmt.h>
+#include <zrepl_mgmt.h>
+#include <uzfs_io.h>
+#include <system_error>
+
+#include <gtest/gtest.h>
+
+#define DISK "/tmp/zrepl_test_disk.img"
+#define POOL_NAME "test_export"
+
+int
+create_file(void)
+{
+       int fd;
+
+       if ((fd = open(DISK, O_RDWR | O_CREAT | O_TRUNC, 0666)) < 0)
+               throw std::runtime_error("cant create disk image");
+
+       if (ftruncate(fd, 100 * 1024 * 1024) != 0)
+               throw std::runtime_error("cant truncate disk image");
+       close(fd);
+       return (0);
+}
+
+/*
+ * if you find the need to copy paste the following
+ * two functions consider to put them in a generic
+ * header file, because I already copied them =)
+ *
+ */
+
+std::string getCmdPath(std::string zfsCmd) {
```

Comment:
there seems to be this functions already existing. It would be better to reuse them.

Reply:
these* functions

Reply:
They where replaced very recently by Jan.. for future references yes, we should have a cb_common.hpp or something

Reply:
ok.. makes sense